Frameworks/BlockchainCertificates/Issuer.swift

```diff
@@ -77,12 +80,14 @@ public struct Issuer {
                 email: String,
                 image: Data,
                 id: URL,
-                url: URL) {
+                url: URL,
+                revocationURL: URL?) {
```

Comment:
You can default that last param to nil to avoid changing the old code. `revocationURL: URL? = nil) {`